sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -14,9 +14,10 @@
 - Renamed `GetSharePropertiesResult` to `ShareProperties`.
 - Renamed `GetShareStatisticsResult` to `ShareStatistics`.
 - Renamed `GetShareDirectoryPropertiesResult` to `ShareDirectoryProperties`.
-- Renamed `GetShareFilePropertiesResult` to `ShareFileServiceProperties`.
+- Renamed `GetShareFilePropertiesResult` to `FileServiceProperties`.
```

Comment:
Fix the source as well to match this change. It still returns `ShareFileServiceProperties`
https://github.com/Azure/azure-sdk-for-cpp/blob/a5bc8fe0efa4534a45692c2787642ba1f073a6da/sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_service_client.hpp#L89-L102

Reply:
https://github.com/Azure/azure-sdk-for-cpp/blob/3a703630327b51d18cfa267349bc00b64c102486/sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_service_client.hpp#L100

It's already fixed in this PR

Reply:
I missed it. I must have been looking at an older commit.